ios/Classes/SwiftFlutterWeatherSensorPlugin.swift

```diff
@@ -1,14 +1,76 @@
 import Flutter
 import UIKit
+import Combine
 
-public class SwiftFlutterWeatherSensorPlugin: NSObject, FlutterPlugin {
+@available(iOS 13.0, *)
+public class SwiftFlutterWeatherSensorPlugin: NSObject, FlutterPlugin, FlutterStreamHandler {
+  private var eventSink: FlutterEventSink? = nil
```

Comment:
I am pretty sure you don't have to initialize optional instance vars to nil. I think you'd only initialize them if they needed to be optional but the initial value needs to be something other than nil.